docs/sidebar.yml

```diff
@@ -89,6 +89,8 @@ sidebar:
               url: docs/reference/metaprogramming/tasty-inspect.html
         - title: Other New Features
           subsection:
+            - title: TypeTest
+              url: docs/reference/other-new-features/type-test.html
```

Comment:
I would put this at the bottom of the list. Most users will never have to use this feature.

Reply:
done